src/fetch.js

```diff
@@ -193,11 +192,13 @@ const getPagedData = async (
   responseKey,
   query = {},
   skip = 0,
-  limit = 100,
+  limit = config?.limit || 50,
```

Comment:
you dont need to provide 50 here  as its already configured plugin option schema file as default value please verify and add changes